### DISCUSSION TITLE
Add support for group management

### DISCUSSION THREAD

**Discussion by xoda** [d99kris/nchat#98] [Ideas]
Hello @d99kris ,
I recently started using `nchat` and I'm genuinely impressed. 

One feature that is stopping me to completely abandon any other client is the complete lack of group management functionalities (creation, adding people, making admin, dismissal, quit etc.). 

Do you have these features in your roadmap? 

Thanks for the great work, 
xoda

**Comment by d99kris** [comment]
Hello @xoda - thanks for the positive feedback!

Group management functionalities are not on the roadmap (yet).. It's _possibly_ something I can look into eventually.. Will update here when/if that happens..

**Comment by d99kris** [comment]
I'm reviewing the feature request backlog now, and I think `group management` functionalities is not something I'll realistically ever get to. So I will proceed to close this request.
